rust/Cargo.lock

```diff
@@ -525,7 +525,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "117725a109d387c937a1533ce01b450cbde6b88abceea8473c4d7a85853cda3c"
 dependencies = [
  "lazy_static",
- "windows-sys 0.59.0",
+ "windows-sys 0.52.0",
```

Comment:
Was it intentional to make this change? We still include windows-sys 0.59.0, but a lot of deps use 0.52.0 now. I guess it probably doesn't matter.